improver/psychrometric_calculations/hail_size.py

```diff
@@ -551,34 +405,40 @@ def hail_size_data(
         data where the ccl_temperature is below 268.15K.
 
         Args:
-            temperature_at_268
+            temperature_at_268:
                 Cube of the temperature of the environment at 268.15K
-            pressure_at_268
+            pressure_at_268:
                 Cube of the pressure of the environment at 268.15K
-            ccl_pressure
+            ccl_pressure:
                 Cube of cloud condensation level pressure
-            ccl_temperature
+            ccl_temperature:
                 Cube of cloud condensation level pressure
-            humidity_mixing_ratio_at_268
-                Cube of humidity mixing ratio at the pressure of the environment at 268.15K
-            wet_bulb_zero
+            humidity_mixing_ratio_at_ccl:
+                Array of humidity mixing ratio at the pressure of the environment at the CCL
```

Comment:
I don't think we're interested in the environment humidity at this point. We are interested in the behaviour of surface-driven convection. The CCL is the point at which air lifted from the surface reaches saturation point. The specific humidity of the CCL is therefore the same as at the surface (but I don't have access to that in this plugin). As I know it will be the saturation point, I can rederive the saturation humidity at the CCL and use this to drive saturated adiabatic ascent of surface air above the CCL to the point I need (-5C isotherm).